docs/terms-of-service.rst

```diff
@@ -0,0 +1,656 @@
+Before sending a takedown request, you must consider legal uses such as fair use and licensed uses.
+
+We will terminate the Accounts of repeat infringers of this policy.
+
+
+Intellectual property notice
+----------------------------
+
+**Short version:** *We own the service and all of our content.
+In order for you to use our content, we give you certain rights to it,
+but you may only use our content in the way we have allowed.*
+
+Read the Docs' rights to content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs and our licensors, vendors, agents, and/or our content providers
+retain ownership of all intellectual property rights of any kind related to our Websites and Services.
+We reserve all rights that are not expressly granted to you under this Agreement or by law.
+
+Read the Docs trademarks and logos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you'd like to use Read the Docs's trademarks,
+you must follow all of our `trademark guidelines`_.
+
+.. _trademark guidelines: https://read-the-docs-guidelines.readthedocs-hosted.com/
+
+
+API terms
+---------
+
+**Short version:** *You agree to these Terms of Service,
+plus this Section, when using any of Read the Docs' APIs (Application Provider Interface),
+including use of the API through a third party product that accesses Read the Docs.*
+
+No abuse or overuse of the API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Abuse or excessively frequent requests to Read the Docs via the API may result in the temporary or permanent suspension of your Account's access to the API.
+Read the Docs, in our sole discretion, will determine abuse or excessive usage of the API.
+We will make a reasonable attempt to warn you via email prior to suspension.
+
+You may not share API tokens to exceed Read the Docs' rate limitations.
+
+You may not use the API to download data or Content from Read the Docs for spamming purposes,
+including for the purposes of selling Read the Docs users' personal information, such as to recruiters, headhunters, and job boards.
+
+All use of the Read the Docs API is subject to these Terms of Service and our :doc:`privacy-policy`.
+
+Read the Docs may offer subscription-based access to our API for those Users who require high-throughput access
+or access that would result in resale of Read the Docs' Service.
+
+
+Additional terms for Documentation Sites
+----------------------------------------
+
+**Short version:** *Documentation Sites on Read the Docs are subject to certain rules,
+in addition to the rest of the Terms.*
+
+Documentation Sites
+~~~~~~~~~~~~~~~~~~~
+
+Each Read the Docs Account comes with the ability to host Documentation Sites.
+This hosting service is intended to host static web pages for All Users.
+Documentation Sites are subject to some specific bandwidth and usage limits,
+and may not be appropriate for some high-bandwidth uses or other prohibited uses.
+Read the Docs reserves the right at all times to reclaim any Read the Docs subdomain without liability.
```

Comment:
"Read the Docs subdomain" might not be clear enough here, it's more of an internal concept. Perhaps it's best to use a higher level concept? We're communicating here that we have the right to alter or remove a project at any time. We don't define what "Project" is, but perhaps this is a helpful addition. We can say "we have the right to remove User Generated Content," but this has a slightly different meaning.

Reply:
This is true. GitHub defines subdomains in their [definitions section](https://github.com/github/site-policy/blob/master/Policies/github-terms-of-service.md#a-definitions). We have already say we "have the right to remove content or close Accounts if we need to" in our User-Generated Content section. Maybe this can just be removed.

The core concept that I wanted to get across and that I think GitHub wanted to get across in their terms is that if they need to reclaim something like github.com/store (because there's a user named "store") that they can. The same applies to subdomains (eg. `*.readthedocs.io` for us and `*.github.io` for github).

Reply:
It looks like they don't really defined this sort of scenario in super clear terms either. Perhaps the blob should just clarify what the subdomain is -- "subdomains on readthedocs.io and readthedocs-hosted.com, prefixed with your project name" etc -- but also maybe describe that we have the right to alter project names? For instance, if your project is abandoned, or in conflict with another project, we reserve the right to rename your project.

I only mention the second case here as this is the net effect the user would notice -- the project slug would change. Feel free to reword things, I'm only using examples here. I think we're on the same on what needs to be clarified